### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by jeffakolb**
fix typo in script name
 enforce line length for readability

### CODE REVIEW

**Comment by jrmontag** [comment]
Looking at the rendered README in this PR, I don't think the new line breaks are correct. Currently, there are sentences (and sometimes code) split between headings and normal text. I think it might make more sense to keep e.g. "Goal" in the heading (and remove the colon), and then ensure that the text that follows is just normal text. If it's helpful to review while working on it, consider using a browser plugin to render markdown locally.

**Comment by jeffakolb** [comment]
@jrmontag just committed a fix to the README formatting

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
-# tweet_parser
+# tweet_parser 
 Authors: Fiona Pigott, Jeff Kolb, Josh Montague, Aaron Gonzales
 
-## Goal:
-Allow reliable parsing of Tweets delivered by the Gnip platform, in both activity-streams and original formats. 
+## Goal:
```
Comment (binaryaaron) [inline README.md]: Do we want to leave the colons or... ?

Comment (jrmontag) [inline README.md]: I'm anti-colon. Also, I think bike shed should be green :)

**Comment by jrmontag** [comment]
Nice, thanks @jeffakolb! Looks great. I'm not going to block on the colon/no-colon question that @binaryaaron raised.

**Pull request merged**